<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ogwindow</class>
 <widget class="QDialog" name="ogwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Zur√ºck</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightAnna">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Anna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightSeverin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Severin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightWC">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>WC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightBad">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Bad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightSchrank">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Schrank</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightVorraum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorraum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightStiege">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Stiege</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightSchlaf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Schlaf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightBadSpiegel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Bad Spiegel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>ogwindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
